test/com/amazon/ionelement/demos/java/ConstructionDemo.java

```diff
@@ -0,0 +1,211 @@
+package com.amazon.ionelement.demos.java;
+
+import com.amazon.ion.Decimal;
+import com.amazon.ion.Timestamp;
+import com.amazon.ionelement.api.AnyElement;
+import com.amazon.ionelement.api.ElementLoader;
+import com.amazon.ionelement.api.ElementType;
+import com.amazon.ionelement.api.IntElement;
```

Comment:
nit: some unused imports here. Also `jupiter.api.Test`, and `ionelement.api.Ion.emptyIonList`